package.json

```diff
@@ -87,14 +87,14 @@
     "pretest": "npm run lint",
     "postinstall": "link-parent-bin && npm run compile && npm run bootstrap:hoist",
     "start": "webpack-dev-server",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-prod": "tt-serve --env.disableHotReloading -p",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start-heroku": "npm run terra -- express-server --port $PORT --site './build'",
```

Comment:
does this line work as `terra express-server --port   $PORT --site './build'`?

Reply:
Good catch! https://github.com/cerner/terra-framework/pull/1368/commits/8bca0bfcb0012fd846f1631ca5ddcd05740b59b5